pint/quantity.py

```diff
@@ -64,6 +63,19 @@ def wrapped(self, *args, **kwargs):
         return result
     return wrapped
 
+def check_implemented(f):
+    def wrapped(self, *args, **kwargs):
+        other=args[0]
+        if other.__class__.__name__ in ["PintArray", "Series"]:
+            return NotImplemented
+        # pandas often gets to arrays of quantities [ Q_(1,"m"), Q_(2,"m")]
+        # and expects Quantity * array[Quantity] should return NotImplemented
+        elif type(other)==list and isinstance(other[0], type(self)):
```

Comment:
Same question here, why not `isinstance` here?

Reply:
fixed